src/gcodeExport.cpp

```diff
@@ -602,10 +630,10 @@ void GCodeExport::writeTravel(int x, int y, int z, double speed)
     CommandSocket::sendLineTo(travel_move_type, Point(x, y), display_width);
 
     *output_stream << "G0";
-    writeFXYZE(speed, x, y, z, current_e_value);
+    writeFXYZE(speed, x, y, z, current_e_value, PrintFeatureType::MoveCombing);
```

Comment:
I would rather see the travel time as a part of the last print feature, unless when moving between different print features.

Because infill patterns can severely impact the travel time I think it would be best to count the travel between infill lines as infill time.